cmd/soroban-cli/src/config/locator.rs

```diff
@@ -114,6 +116,20 @@ pub struct Args {
     pub config_dir: Option<PathBuf>,
 }
 
+#[derive(Debug, clap::Args, Default, Clone)]
+#[group(skip)]
+#[cfg(not(feature = "version_lt_23"))]
+#[cfg(feature = "version_gte_23")]
+pub struct Args {
+    /// ⚠️ Deprecated: global config is always on
+    #[arg(long, global = true, help_heading = HEADING_GLOBAL)]
+    pub global: bool,
+
+    /// Location of config directory, default is "$`XDG_CONFIG_HOME/.stellar`"
```

Comment:
Why the backticks in the path? I would have thought:
```suggestion
    /// Location of config directory, default is "$XDG_CONFIG_HOME/.stellar"
```

Reply:
Fixed!